scripts/srv_git_update.sh

```diff
@@ -17,6 +17,10 @@ count=`git rev-list master ^origin/master --count`
 if [ "$count" -ne "0" ]; then	# 5. There will be updates
 	# 5a Update the local repo
 	git pull origin master
-	# 5b Update the SHA256 hash table
+	# 5b Do rsync of files than may have changed to data/cache
+	rsync -a cache ../data
```

Comment:
If a file is deleted from the gmtserver-admin/cache directory, we also need to delete it from the data/cache directory. Thus `--delete` is needed.
```suggestion
	rsync -a --delete cache ../data
```